.github/workflows/checks.yml

```diff
@@ -203,6 +203,7 @@ jobs:
 
       - name: Run licensei
         env:
-           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This space change is purely stylistic.

Reply:
Can remove this change if desired.

Reply:
Removed this change to verify https://github.com/opentffoundation/opentf/pull/408#discussion_r1325947154

Reply:
This is *what* fixes the CI :facepalm: ...